mvc/3.0/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Jakarta MVC 3.0 (Under Development)"
+title: "Jakarta MVC 3.0"
 date: 2023-05-17
 summary: "Release supporing Jakarta EE 11"
```

Comment:
MVC 3.0 is not included in Jakarta EE 11.

Reply:
True! The description "supporting" is to show that it runs on a Jakarta EE 11 implementation. Maybe change the wording to requires? It may work with some EE10 implementations, but not likely.

Reply:
Personally, I think it's better to change the wording. Normally, "Jakarta EE 11" refers to the name of the specification, and if it is intended as an implementation, it should be described differently.

Reply:
Possibly change to "aligned with"?